build.gradle

```diff
@@ -44,14 +44,20 @@ repositories {
     mavenCentral()
 }
 
+configurations {
+    provided
+}
+
 dependencies {
     compile("com.emc.ecs:object-client:3.0.1") {
         exclude group: "org.slf4j", module: "slf4j-api"
         exclude group: "org.slf4j", module: "slf4j-log4j12"
     }
 
-    compileOnly "org.apache.spark:spark-core_$versions.scala:$versions.spark",
-                "org.apache.spark:spark-sql_$versions.scala:$versions.spark"
+    provided "org.apache.spark:spark-core_$versions.scala:$versions.spark",
+             "org.apache.spark:spark-sql_$versions.scala:$versions.spark"
+
+    compileOnly configurations.provided
```

Comment:
It is there to explicitly control what goes into the "provided" scope in the pom.  But if that matches exactly with the "compileOnly" scope in gradle, then maybe we don't need it (now that joda-time is gone).  Shall I revert it?